.github/workflows/build-prod-app.yml

```
@@ -0,0 +1,66 @@
+name: "Prod build"
+
+on:
+  push:
+    branches:
+      - prod
+  pull_request:
+    branches:
+      - prod
+    types: [ opened, synchronize ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  code-analysis:
+    uses: ./.github/workflows/code-analysis.yml
+  unit-tests:
+    uses: ./.github/workflows/gradle-run-unit-tests.yml
+  build-app:
+    needs: [ unit-tests, code-analysis ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive # Needed in order to fetch Kalium sources for building
+          fetch-depth: 0
+      - name: Set up JDK 17
+        uses: buildjet/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: gradle
+      - name: Decode Keystore
+        env:
+          ENCODED_STRING: ${{ secrets.ENCODED_KEYSTORE_PUBLIC_RELEASE }}
+        run: |
+          TMP_KEYSTORE_FILE_PATH="${RUNNER_TEMP}"/keystore
+          mkdir "${TMP_KEYSTORE_FILE_PATH}"
+          echo $ENCODED_STRING | base64 -di > "${TMP_KEYSTORE_FILE_PATH}"/key.keystore
+      - name: Make gradlew executable
+        run: chmod +x ./gradlew
+      - name: build prod flavour APK
+        run:
+          ./gradlew app:assembleProdCompatRelease -p ./ --no-daemon
```

```suggestion
          ./gradlew app:assembleProdCompatRelease
```

Not sure there's any benefit in `--no-daemon`. On the contrary, running multiple builds on separate Gradle processes one after the other should be slower, as Gradle will need to restart itself again and again.

Same for all the other files and runs.

----

`-p ./` is redundant, isn't it?